### PR TITLE
feat: Support skipping specific checks with an environment variable(#601)

### PR DESCRIPTION
**Pull request by YeswanthRajakumar**
#### ✔ Checklist:

- [x] Commit messages have been written in [Conventional Commits](https://www.conventionalcommits.org) format;
- [x] Tests for the changes have been added (for bug fixes / features);
- [ ] Docs have been added / updated on the [dotenv-linter.github.io](https://github.com/dotenv-linter/dotenv-linter.github.io) (for bug fixes / features).

### CODE REVIEW

File: src/cli/options.rs [hunk 1]
```diff
@@ -74,3 +69,44 @@ fn get_many_from_args<'a, T: Clone + Send + Sync + 'static>(
...
+        .into_iter()
+        .map(|l| l.to_owned())
+        .collect();
+
+    get_skip_lints_from_env_vars(&mut skip);
+    skip
+}
+
+fn get_skip_lints_from_env_vars(skip: &mut Vec<LintKind>) {
+    let env_skip_vars = env::var("DOTENV_LINTER_SKIP");
```
Comment (DDtKey) [inline src/cli/options.rs:84]: You don't need to do it manually, `clap` has embedded support
https://docs.rs/clap/latest/clap/builder/struct.Arg.html#method.env

**Review by DDtKey — CHANGES_REQUESTED** [review]

File: src/cli/options.rs (hunk 1 above)
Comment (YeswanthRajakumar) [reply src/cli/options.rs]: I've replaced my logic with clap env

**Review by DDtKey — CHANGES_REQUESTED** [review]

File: src/cli/mod.rs [hunk 2]
```diff
@@ -100,6 +100,13 @@ fn fix_command() -> Command {
 
 fn common_args() -> Vec<Arg> {
     vec![
+        Arg::new("env")
+            .long("flag")
+            .env("DOTENV_LINTER_SKIP")
+            .action(ArgAction::Set)
+            .num_args(1..)
+            .value_delimiter(',')
+            .value_parser(value_parser!(LintKind)),
```
[the remark this answers is not in the document]
Comment (YeswanthRajakumar) [reply src/cli/mod.rs:109]: Cool, will do it

**Comment by DDtKey** [comment]
Could you, please, rebase your branch on current main of `dotenv-linter`?

**Review by DDtKey — APPROVED** [review]
Thank you for your contribution!

Update the branch & LGTM.

It would be nice to add a new `ENV` flag to the documentation: https://github.com/dotenv-linter/dotenv-linter.github.io

**Pull request closed without merging**